credly/views.py

```diff
@@ -345,10 +345,16 @@ def get_mozilla_badges(credly=None, access_token=None):
     if not access_token:
         credly = get_our_credly()
         access_token = get_our_access_token()
-        verbose = 0
         show_earned = 0
-    list_result = credly.members(MozillaAccountId).badges.created.get(verbose=1, show_earned=1, access_token=access_token)
-    return list_result['data']
+
+    headers = XAPIHeaders()
+    headers['X-Allow-HTML'] = 1
```

Comment:
yep. it's literally the number 1 - Credly made an interesting choice about header values =)